Content.Server/StationEvents/Events/PowerGridCheckRule.cs

```diff
@@ -19,17 +17,25 @@ public sealed class PowerGridCheckRule : StationEventSystem<PowerGridCheckRuleCo
     {
         [Dependency] private readonly ApcSystem _apcSystem = default!;
 
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<PowerGridCheckDisabledComponent, ApcToggleMainBreakerAttemptEvent>(OnApcToggleMainBreaker);
+        }
+
         protected override void Started(EntityUid uid, PowerGridCheckRuleComponent component, GameRuleComponent gameRule, GameRuleStartedEvent args)
         {
             base.Started(uid, component, gameRule, args);
 
             if (!TryGetRandomStation(out var chosenStation))
                 return;
 
-            foreach (var (apc, transform) in EntityQuery<ApcComponent, TransformComponent>(true))
+            var query = EntityQueryEnumerator<ApcComponent, TransformComponent>();
```

Comment:
AllEntityQuery

Reply:
why should it be applied to paused entities?